internal/local/command_kafka_start.go

```diff
@@ -29,6 +29,7 @@ import (
 )
 
 const (
+	dockerWorkingVersionMsg   = "The local commands have been verified to work with Docker Desktop version as late as 4.25.0. Verify that you are not using an unverified version of Docker (versions greater than v4.25.0)."
```

Comment:
```suggestion
	dockerWorkingVersionMsg   = "The local commands have been verified to work with Docker Desktop version as late as 4.25.0. Verify that you are not using an unverified version of Docker."
```

The first part of the string already says which versions are verified, so I think we can drop the last bit.